js/sr_tools_db.js

```diff
@@ -1722,91 +1731,629 @@ var db = {
-				res.max_attributes.will = 7;
+				res.attributes.body += 2;
+				res.min_attributes.body += 2;
+				res.max_attributes.body += 2;
+				res.attributes.reaction += -1;
+				res.max_attributes.reaction += -1;
+				res.attributes.strength += 2;
+				res.min_attributes.strength += 2;
+				res.max_attributes.strength += 2;
+				res.attributes.will += 1;
+				res.min_attributes.will += 1;
+				res.max_attributes.will += 1;
+				res.augmentations.push({name: 'Thermographic Vision'});
+				res.augmentations.push({name: 'Pathogen Resistance (2)'});
+				res.augmentations.push({name: 'Toxin Resistance (2)'});
+				res.augmentations.push({name: 'Increased Lifestyle Costs (120%)'});
 				break;
 
 			case 'Ork':
-				res.attributes.body = 3;
-				res.min_attributes.body = 4;
-				res.max_attributes.body = 9;
-				res.attributes.strength = 2;
-				res.min_attributes.strength = 3;
-				res.max_attributes.strength = 8;
-				res.attributes.logic = -1;
-				res.min_attributes.logic = 1;
-				res.max_attributes.logic = 5;
-				res.attributes.charisma = -1;
-				res.min_attributes.charisma = 1;
-				res.max_attributes.charisma = 5;
+				res.attributes.body += 3;
+				res.min_attributes.body += 3;
+				res.max_attributes.body += 3;
+				res.attributes.strength += 2;
+				res.min_attributes.strength += 2;
+				res.max_attributes.strength += 2;
+				res.attributes.logic += -1;
+				res.max_attributes.logic += -1;
+				res.attributes.charisma += -1;
+				res.max_attributes.charisma += -1;
+				res.augmentations.push({name: 'Low-Light Vision'});
 				break;
 
 			case 'Troll':
-				res.attributes.body = 4;
-				res.min_attributes.body = 5;
-				res.max_attributes.body = 10;
-				res.attributes.agility = -1;
-				res.min_attributes.agility = 1;
-				res.max_attributes.agility = 5;
-				res.attributes.strength = 4;
-				res.min_attributes.strength = 5;
-				res.max_attributes.strength = 10;
-				res.attributes.logic = -1;
-				res.min_attributes.logic = 1;
-				res.max_attributes.logic = 5;
-				res.attributes.intuition = -1;
-				res.min_attributes.intuition = 1;
-				res.max_attributes.intuition = 5;
-				res.attributes.charisma = -2;
-				res.min_attributes.charisma = 1;
-				res.max_attributes.charisma = 4;
-				res.augmentations.push({name: 'Troll Dermal Deposits'});
+				res.attributes.body += 4;
+				res.min_attributes.body += 4;
+				res.max_attributes.body += 4;
+				res.attributes.agility += -1;
+				res.max_attributes.agility += -1;
+				res.attributes.strength += 4;
+				res.min_attributes.strength += 4;
+				res.max_attributes.strength += 4;
+				res.attributes.logic += -1;
+				res.max_attributes.logic += -1;
+				res.attributes.intuition += -1;
+				res.max_attributes.intuition += -1;
+				res.attributes.charisma += -2;
+				res.max_attributes.charisma += -2;
+				res.augmentations.push({name: 'Thermographic Vision'});
+				res.augmentations.push({name: 'Extended Reach (1)'});
+				res.augmentations.push({name: 'Troll Dermal Deposits (1)'});
```

Comment:
Changing the name of the Troll's natural hide here will break some of the calculations in the `sr_tools_render.js` file. Please look at these functions for where the string "Troll Dermal Deposits" is being used:

- save_mook()
- mook_for_action()
- mook_for_display()

Reply:
Changed all strings accordingly in new commit